Web/package.json

```diff
@@ -33,7 +33,7 @@
     "rxjs": "^6.5.3",
     "seedrandom": "^3.0.5",
     "smoothscroll-polyfill": "^0.4.4",
-    "styled-components": "^4.4.1",
+    "styled-components": "^5.0.0-rc.1",
```

Comment:
isn't styled-component v5 `beta`? is it safe to use it?

Reply:
Hi there! styled-components v5 seems to be in release candidate now, which seems to indicate to me that they are very close to release (assuming a release cycle of alpha -> beta -> release candidate -> release).

We have been using this for our app, and so far nothing breaks!